demos/react/ecommerce/src/App.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import algoliasearch from 'algoliasearch/lite'
+import {
+  Configure,
+  InstantSearch,
+  Pagination,
+  RangeInput,
+  RefinementList,
+  SearchBox,
+} from 'react-instantsearch-hooks-web'
+
+import { Content } from './Content'
+import { Stats } from './Stats'
```

Comment:
stats exists in React InstantSearch Hooks

Reply:
Oh I didn't know that, we only document the [hook `useStats`](https://www.algolia.com/doc/api-reference/widgets/stats/react-hooks/) for now, I think we should document the widget instead, WDYT?

Reply:
yep, we should document the widget too I think

Reply:
I'll create two tickets for RISH, one to document the `Stats` widget, and one to expose the `useGeoSearch` hook.